tests/foreman/ui/test_organization.py

```diff
@@ -289,3 +329,74 @@ def test_positive_download_debug_cert_after_refresh(session):
                 session.subscription.refresh_manifest()
     finally:
         entities.Subscription(organization=org).delete_manifest(data={'organization_id': org.id})
+
+
+@pytest.mark.tier2
+def test_negative_errata_view_organization_switch(
+    session, module_org, module_lce, module_repos_col, request, default_sat
+):
+    """Verify no errata list visible on Organization switch
+
+    :id: a5317b2d-2593-4bee-9b20-78066cf2964c
+
+    :Steps: Create an Organization having a product synced which contains errata.
+
+    :expectedresults: Verify that the errata belonging to one Organization is not
+                      showing in the Default organization.
+
+    :CaseImportance: Critical
+
+    :CaseLevel: Integration
+    """
+    rc = RepositoryCollection(repositories=[YumRepository(settings.repos.yum_3.url)])
+    rc.setup_content(module_org.id, module_lce.id)
+    with session:
+        assert (
+            session.errata.search(CUSTOM_REPO_ERRATA_ID, applicable=False)[0]['Errata ID']
+            == CUSTOM_REPO_ERRATA_ID
+        )
+        session.organization.select(org_name="Default Organization")
+        assert not session.errata.search(CUSTOM_REPO_ERRATA_ID, applicable=False)
+
+
+@pytest.mark.tier2
+@pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
+def test_negative_product_view_organization_switch(session, module_org):
+    """Verify product created in one organization is not visible in another
+
+    :id: 47f3580e-6071-4499-bf27-1ded62a7d3e3
+
+    :Steps:
+            1. Create an Organization having a product and verify that product is present in
+               the Organization.
+            2. Switch the Organization to default and verify that product is not visible in it.
+
+
+    :expectedresults: Verify that the Product belonging to one Organization is not visible in
+                      another organization.
+
+    :CaseLevel: Integration
```

Comment:
Is there a BZ associated with this test coverage?

Also, for metadata, if the test case *is* negative (see other comment), it needs a testimony token to set that metadata:
https://github.com/SatelliteQE/robottelo/blob/master/testimony.yaml#L130

The default is positive.

Reply:
Updated test as positive. It doesn't have any BZ associated.